packages/rn-tester/e2e/__tests__/Alert-test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+/* global device, element, by, expect, waitFor */
+const {openExampleWithTitle} = require('../e2e-helpers');
+
+describe('Alert', () => {
+  beforeAll(async () => {
+    await element(by.id('apis-tab')).tap();
+    await element(by.id('explorer_search')).replaceText('Alert');
+    await element(by.label('Alerts')).tap();
+  });
+
+  afterAll(async () => {
+    await element(by.label('Back')).tap();
+  });
+
+  it('should show alert dialog with message and default button', async () => {
```

Comment:
Ideally these test descriptions would specify which example they're testing.

Reply:
I am not sure how I should add them. I have pushed one commit. Please let me know if that's the correct way to do it.